suzieq/poller/worker/services/interfaces.py

```diff
@@ -824,21 +836,30 @@ def _clean_iosxr_data(self, processed_data, raw_data):
             elif devtype == 'iosxr':
                 entry['ip6AddressList'] = []
 
+            if entry['ifname'].startswith('Vlan'):
+                iftype = 'vlan'
+                if entry.get('vlan', '') == '':
+                    entry['vlan'] = entry['ifname'].split('Vlan')[1].strip()
+            if entry.get('vlan', '') and entry.get('innerVlan', ''):
+                iftype = "qinq"
+
+            entry['type'] = iftype
             # This is specifically for IOSXE/IOS devices where
             # the IPv6 address uses capital letters
             if devtype != 'iosxr':
                 entry['ip6AddressList'] = [x.lower()
                                            for x in entry.get('ip6AddressList',
                                                               [])]
 
+            if entry['ifname'].endswith('.0'):
+                entry['vlan'] = -1
```

Comment:
Use a constant defined inside utils instead of '-1'?

Reply:
Ack, I agree